### PR TITLE
List shard support

### PR DESCRIPTION
**Pull request by kailashhd**

### CODE REVIEW

File: actions/listShards.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+module.exports = function listShards(store, data, cb) {
+  // https://docs.aws.amazon.com/kinesis/latest/APIReference/API_ListShards.html
+  // Currently we don't support the ExclusiveStartShardId and NextToken requests.
+  store.getStream(data.StreamName, function(err, stream) {
+    if (err) return cb(err)
+
+    cb(null, {Shards: stream.Shards})
```
Comment (smthomas1704) [inline actions/listShards.js:8]: Does cb() only have a side effect and no return value? Do we need to return the value here?

File: test/listShards.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+              SequenceNumberRange: {},
+              HashKeyRange: {
+                StartingHashKey: '0',
+                EndingHashKey: '113427455640312821154458202477256070484',
+              },
+            }, {
+              ShardId: 'shardId-000000000001',
+              SequenceNumberRange: {},
+              HashKeyRange: {
+                StartingHashKey: '113427455640312821154458202477256070485',
```
Comment (smthomas1704) [inline test/listShards.js:100]: Are these values mocked somewhere? Will they change ever?

**Review by smthomas1704 — APPROVED** [review]

File: test/listShards.js (hunk 2 above)
Comment (kailashhd) [reply test/listShards.js:100]: The number of shards is stored in ./helper.js. The hash key is an implementation details and is guaranteed to be those values. (Hash values is typically @^64 or 2^128 / number of shards).

File: actions/listShards.js (hunk 1 above)
Comment (kailashhd) [reply actions/listShards.js:8]: According to the this place where it is called from https://goo.gl/36Xw6n, the 2 parameters for the function is error and data. Correct me if my understanding is wrong, but that will ensure that error is null and return values is data. (in JSON format).

**Comment by kailashhd** [comment]
Closing this. Not necessary anymore.

**Pull request closed without merging**